tesseract_collision/include/tesseract_collision/bullet/bullet_utils.h

```diff
@@ -159,7 +159,7 @@ class CollisionObjectWrapper : public btCollisionObject
   {
     m_data.push_back(t);
   }
-
+  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```

Comment:
Does it matter where it goes in the class/struct?

Reply:
I don't think so. The documentation seems to imply that as long as it is public it should be fine. I can move it to the top if you prefer though.